.npmignore

```diff
@@ -0,0 +1,7 @@
+.github
+.eslintrc.js
+.eslintignore
+.gitignore
+.nvmrc
+.editorconfig
```

Comment:
`.gitignore` is always ignored by default.
Also, let's sort things alphabetically, and indicate what is a directory.

```suggestion
.github/
.editorconfig
.eslintignore
.eslintrc.js
.nvmrc
```